ivadomed/utils.py

```diff
@@ -745,3 +809,95 @@ def save_feature_map(batch, layer_name, context, model, test_input, slice_axis):
         nib_pred = nib.Nifti1Image(attention_map, nib_ref.affine)
 
         nib.save(nib_pred, save_directory)
+
+
+def save_color_labels(gt_data, binarize, gt_filename, output_filename, slice_axis):
+    rdict = {}
```

Comment:
Saves .nii.gz with color labels which helps identify the different labels compared to grayscales (especially if there is a lot of labels)
![image](https://user-images.githubusercontent.com/49137243/77611250-33047c00-6efb-11ea-9ba8-f995961f5737.png)

Reply:
Really cool! :smiley: